src/contexts/DocumentListContext.jsx

```diff
@@ -50,17 +53,17 @@ export const DocumentListContextProvider = ({ children }) => {
       setDocumentListObject({
         docList: [],
         dataset: createSolidDataset(),
-        containerUrl: `${selectedUser.podUrl}PASS/Documents/`
+        containerUrl: client ? `${client?.podUrl}PASS/Documents/` : `${podUrl}PASS/Documents/`
```

Comment:
if `client` is `true` do we need the `?` guard on `${client.podUrl}PASS/Documents/` ?